ingest/validation/validate_metadata.py

```diff
@@ -234,39 +235,39 @@ def process_metadata_row(metadata, convention, line):
         if k in metadata.type['convention']['integer']:
             try:
                 row_info[k] = int(v)
-            except ValueError as e:
-                error_msg = f'ERROR: value provided does not match convention type delaration for {k}'
+            except ValueError:
```

Comment:
Exception handling looks identical across all three cases here.

Moving the `try` just before and the `except` just after the `if k...` block would eliminate two extraneous `try`/`except` blocks.

Reply:
I wondered what I could do to make the code cleaner but didn't see a solution - many thanks for the suggestion!

Reply:
updated in `4023a08`